README.md

```diff
@@ -90,6 +95,9 @@ select a SBAS for a given location on Earth.
 * `--flate2`  
 allow native parsing of .gz compressed RINEX files. Otherwise, user must uncompress manually the `.gz` extension first.
 
+* `--pyo3` (experimental)
+Add Python bindings via `PyO3`. To build the Python package, you must first install maturin and then build it with the pyo3 feature flag. For example, `maturin build -F pyo3`. Maturin will then build and place the resulting .whl file in `/target/wheels/`, after which you can install the package with `pip install rinex`.
```

Comment:
It would be awesome to be able to auto generate the wheel

Reply:
So it gets included with the release, and later uploaded to pypi.org ?

Reply:
> So it gets included with the release

yes, like the application binaries

> and later uploaded to pypi.org ?

i did not think about going this far, but I don't foresee any reasons that would not be doable.

Reply:
Ok so [maturin-action](https://github.com/PyO3/maturin-action) does exist,
i just pushed a commit that integrates the basis. I simply reworked a basic example, we need to test it